geopandas/tools/crs.py

```diff
@@ -1,9 +1,9 @@
-import warnings
-
 from pyproj import CRS
```

Comment:
The following one would be better.

```python
import warnings
from typing import Union

from pyproj import CRS
```

std lib should be before 3rd lib.
`import` should be before `from`.

Reply:
Ok. I forgot that warnings was standard lib. I'll change that right now.